homeassistant/components/xiaomi_aqara/strings.json

```diff
@@ -4,9 +4,11 @@
     "step": {
       "user": {
         "title": "Xiaomi Aqara Gateway",
-        "description": "Connect to your Xiaomi Aqara Gateway",
+        "description": "Connect to your Xiaomi Aqara Gateway, if the IP and mac addresses are left empty, auto-discovery is used",
```

Comment:
Instead of having this step, we should do discovery first, and then show a drop down of found entries with "Manual" being one of the options.

Reply:
We can do this in a future PR.

Reply:
Thanks for the hint, that indeed sounds like a nice improvement.
There already is a drop down list in case multiple gateways are discovered using discovery so schould not be to hard to implement this.

I will not have time in the near future to do this, but I will keep it in mind to do it sometime in the future.